### PR TITLE
workflows: tighten regexes for labelling

### PR DESCRIPTION
**Pull request by dawidd6**
Make content regexes tighter, so we don't match on comments or something else.

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: .github/workflows/autolabel.yml [hunk 1]
```diff
@@ -27,14 +27,14 @@ jobs:
...
                 }, {
                     "label": "formula deprecated",
                     "path": "Formula/.+",
-                    "content": "deprecate!"
+                    "content": "\n  deprecate!.*\n"
                 }, {
                     "label": "formula disabled",
                     "path": "Formula/.+",
-                    "content": "disable!"
+                    "content": "\n  disable!\n"
```
Comment (SeekingMeaning) [inline .github/workflows/autolabel.yml:38]:
```suggestion
                    "content": "^  disable!$"
```

Comment (dawidd6) [reply .github/workflows/autolabel.yml:38]: That doesn't work, as we are matching against whole file, not single line.

Comment (SeekingMeaning) [reply .github/workflows/autolabel.yml:38]: Ah okay

**Comment by MikeMcQuaid** [comment]
👍🏻

**Pull request merged**